docs/source/model_definition.md

```diff
@@ -93,13 +152,14 @@ head = Head(
 model = Model(head)
 ```
 
-### Tying embeddings
-For `NextItemPredictionTask` we have added a best practice, **Tying Embeddings**, proposed originally by the NLP community to tie the weights of the input (item id) embedding matrix with the output projection layer. Not only do tied embeddings reduce the memory requirements significantly, but our own experimentation during [recent competitions](https://resources.nvidia.com/en-us-merlin/recommendation-syste?lx=97GH0Q) and empirical analysis detailed in our [paper](https://dl.acm.org/doi/10.1145/3460231.3474255) and [online appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md) show how effective this method is.  It is enabled by default, but can be disabled by setting weight_tying=False).
+### Tying Embeddings
 
+For the {class}`NextItemPredictionTask <transformers4rec.torch.NextItemPredictionTask>` class, we recommend *tying embeddings*.
+The tying embeddings concept was initially proposed by the NLP community to tie the weights of the input (item ID) embedding matrix with the output projection layer.
+Not only do tied embeddings reduce the memory requirements significantly, but our own experimentation during [recent competitions](https://resources.nvidia.com/en-us-merlin/recommendation-syste?lx=97GH0Q) and empirical analysis detailed in our [Transformers4Rec: Bridging the Gap between NLP and Sequential / Session-Based Recommendation](https://dl.acm.org/doi/10.1145/3460231.3474255) paper and online [appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md) demonstrate that this method is very effective.
+Tying embeddings is enabled by default, but can be disabled by setting `weight_tying` to `False`.
```

Comment:
I checked the source code ([here](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/776af7bad969fb408e623d1a3b832e8d5ee337a4/transformers4rec/torch/model/prediction_task.py#L153)), and it seems the default is `False`. But it makes sense to me to set the default to `True` because this method was shown to be very effective. I will try to open a PR changing the default `weight_tying` to True by the end of week.

Reply:
Reading the code--what an incredibly simple way to verify!  I'm sorry that it didn't occur to me to verify it.  Thanks very much.